### PR TITLE
AC Disable Test - LRAC-11314 | master

### PR DESCRIPTION
**Pull request by calvinctvo**
https://issues.liferay.com/browse/LRAC-11314 (ticket)

### CODE REVIEW

**Comment by calvinctvo** [comment]
ci:test:sf

**Comment by manuferreira** [comment]
Hey @calvinctvo I'm taking a look here, one of the test steps is to change the timezone to match de Japan time, the test seems to switch to this timezone without problems. But later in the next steps, the test tries to change to the China timezone but without success, according to the images provided in the [poshi report 
](https://testray.liferay.com/reports/production/logs/2022-06/test-1-27/test-portal-testsuite-upstream(master)/1050/functional-tomcat90-mysql57-jdk8/4/12/LocalFile.Workspace_ViewOnlyOneInformationBannerShownAfterChangeTimezone/index.html.gz?authuser=0) the test does not seem to change to the China timezone, I don't know if it's because of the banner "Welcome to Analytics Cloud", I don't know if I'm correct, but what do you think?

**Comment by christianstokes** [comment]
Hi @calvinctvo can you double check this is not a bug or test fix, then let us know?

**Comment by ViniLopes20** [comment]
Yes, I think what @manuferreira said is a good idea if we did a step in the test to close the modal we would have a better screenshot of the message. By the result of Testray the test is passing in the last 2 builds in the release routine. From the error, it looks like the message was not updated after changing to the China timezone.

**Comment by calvinctvo** [comment]
@christianstokes this is definitely not a bug, since I've manually change the timezone and the banner changes as expected. However, as @ViniLopes20 stated, the message was not updated while running the automated test. The test is flaky recently, could possibly because of the changes with the popup modal.

**Comment by calvinctvo** [comment]
ci:test:sf

**Comment by calvinctvo** [comment]
ci:test:sf

**Comment by ViniLopes20** [comment]
Hey @calvinctvo, I think you could run this test in CI for us to check if the message is being modified or not. I think that way we can send a PR with this fix or deactivate the test if there is a problem in the CI. What do you think? You can commit a change like this to test on CI
LRAC-11341-by-ViniLopes20-·-Pull-Request-119-·-ViniLopes20-liferay-portal
:

**Comment by ViniLopes20** [comment]
ci:test:analytics-cloud:nocompile

**Comment by calvinctvo** [comment]
hey @ViniLopes20 this is the run I tried on upstream: https://testray.liferay.com/home/-/testray/case_results?testrayBuildId=1874012460, I think you're right. The closeModal did close the banner for the previous run we tried: https://testray.liferay.com/home/-/testray/case_results?testrayBuildId=1873638065. So I think this fix should fix this issue.

**Comment by ViniLopes20** [comment]
> hey @ViniLopes20 this is the run I tried on upstream: https://testray.liferay.com/home/-/testray/case_results?testrayBuildId=1874012460, I think you're right. The closeModal did close the banner for the previous run we tried: https://testray.liferay.com/home/-/testray/case_results?testrayBuildId=1873638065. So I think this fix should fix this issue.

Hey @calvinctvo, so the fix that you are going to send to the CI will put it to close the modal, right?

**Comment by calvinctvo** [comment]
> 

Yes, the change in the PR already have the close modal

File: test.properties [hunk 1]
```diff
@@ -8414,4 +8414,4 @@
     vm.cluster-node-2.ip=
 
     vm.db.host=vm-3
-    vm.db.ip=
\ No newline at end of file
+    vm.db.ip=
```
Comment (ViniLopes20) [inline test.properties]: Hey @calvinctvo, could you discard this change?

Comment (calvinctvo) [reply test.properties]: done! I just revert back to this commit 432fba8e2dda42a032aa8abd9144d1fa7662a581

**Review by ViniLopes20 — APPROVED** [review]
Congratulations on the investigation for this fix, Calvin. LGTM! 🎊

**Comment by calvinctvo** [comment]
ci:forward

**Pull request closed without merging**